public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json

```
@@ -0,0 +1,1904 @@
+            "resource": "/subscriptions/$sub",
+            "resultFormat": "table",
+            "workspace": ""
+          },
+          "azureMonitor": {
+            "aggOptions": [],
+            "dimensionFilter": "*",
+            "dimensionFilters": [],
+            "timeGrain": "auto",
+            "timeGrains": [],
+            "top": "10"
+          },
+          "azureResourceGraph": {
+            "resultFormat": "table"
+          },
+          "insightsAnalytics": {
+            "query": "",
+            "resultFormat": "time_series"
+          },
+          "queryType": "Azure Log Analytics",
+          "refId": "A",
+          "subscription": "",
+          "subscriptions": []
+        }
+      ],
+      "title": "${agg:text} Bytes Sent Rate",
+      "transformations": [
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {
+              "Computer": false,
+              "ComputerId": true,
+              "ComputerId1": true,
+              "P5th": true,
+              "P95th": true,
+              "score": true
+            },
+            "indexByName": {},
+            "renameByName": {}
+          }
+        },
+        {
+          "id": "prepareTimeSeries",
+          "options": {
+            "format": "many"
+          }
+        },
+        {
+          "id": "renameByRegex",
+          "options": {
+            "regex": "(.+)\\s(.+)",
+            "renamePattern": "$2"
+          }
+        }
+      ],
+      "type": "timeseries"
+    },
+    {
+      "datasource": "${ds}",
+      "description": "The resource path for this panel uses multiple template variables which requires modifying the dashboard JSON directly. If you would like to do something similar please go to Settings > Save As. Edit as you'd like in your new copy by going to Settings > JSON Model.",
```

gosh there are so many of these lol, maybe we should just put it in the help text at the top instead? I'm open to whatever you think is best, I'm going to stop trying to find all these, but I think you get the gist, I'm just trying to make sure that the steps we're saying suggest the text before
```suggestion
      "description": "The resource path for this panel uses multiple template variables which requires modifying the dashboard JSON directly. If you would like to do something similar please go to Settings > JSON Model. Edit as you'd like in your new copy by going to Settings > Save As.",
```